src/mods/text.js

```diff
@@ -130,7 +132,12 @@ idrinth.text = {
                 enableFavRequest: 'Enable Auto-Raid-Request for Favorites?',
                 favIdToJoin: 'FavoriteIds to join (separate multiple by comma)',
                 worldserver: 'Worldserver?',
-                disableAutoJoinSpecific: 'Disable Autojoining for specific raids'
+                disableAutoJoinSpecific: 'Disable Autojoining for specific raids',
+                chat:{
```

Comment:
try notification here instead of duplicating it below. You might also want to move the whole group into data.chat